.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+body:
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Environment
+      description: Where are you using our software?
+      options:
+        - SaaS (https://sentry.io/)
+        - single-tenant (if you know, you know)
+        - self-hosted (`onpremise` deployment)
```

Comment:
```suggestion
        - self-hosted ([`onpremise`](https://github.com/getsentry/onpremise) deployment)
```

Reply:
Sadly this is not markdown, just _seems_ like it with the backticks. :-/